src/sentry/shared_integrations/client.py

```diff
@@ -44,9 +44,11 @@ def rel(self):
 
     @classmethod
     def from_response(self, response, allow_text=False):
+        if response.request.method == "HEAD":
+            return TextApiResponse(response.status_code)
```

Comment:
is this going to work with our transactions if we try to do `resp.status_code`? I think the `status_code` will just be `None` right?